var/spack/repos/builtin/packages/libmodbus/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install libmodbus
+#
+# You can edit this file again by typing:
+#
+#     spack edit libmodbus
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
```

Comment:
You can delete this stuff, it's only there for new users to create their first package.

Reply:
Oh right did not see that, thanks!